doc/source/rllib/package_ref/policy.rst

```diff
@@ -0,0 +1,32 @@
+.. _policy-reference-docs:
+
+Policy APIs
+===========
+
+.. toctree::
+   :maxdepth: 1
+
+   policy/policy.rst
+   policy/tf_policies.rst
+   policy/torch_policy.rst
+   policy/custom_policies.rst
+
+
+The ``Policy`` class contains functionality to compute actions for decision making
+in an environment, as well as computing loss(es) and gradients, updating a neural
+network model as well as postprocessing a collected environment trajectory.
+One or more ``Policy`` objects sit inside a
+`RolloutWorker's <https://docs.ray.io/en/master/rllib-package-ref.html#module-ray.rllib.evaluation>`_
```

Comment:
1. Shouldn't this link to RolloutWorker the object in particular?
2. We should not have *any* references to master. All references need to link to the current documentation, or else these links easily get stale.

Reply:
(see comment on PR)

Reply:
done